src/jit/gentree.cpp

```diff
@@ -16986,7 +16986,7 @@ regNumber ReturnTypeDesc::GetABIReturnReg(unsigned idx)
         }
     }
 
-#elif defined(_TARGET_X86_) || defined(_TARGET_ARM_)
+#elif defined(_TARGET_X86_)
 
```

Comment:
Can you delete this part of the function comment:
```
// Notes:
//     Right now this is implemented only for x64 Unix
//     and yet to be implemented for other multi-reg return
//     targets (Arm64/Arm32/x86).
//
// TODO-ARM:   Implement this routine to support HFA returns.
// TODO-X86:   Implement this routine to support long returns.
```
since it seems to be obsolete.